plugins/outputs/file/file.go

```diff
@@ -76,19 +97,19 @@ func (f *File) Description() string {
 
 func (f *File) Write(metrics []telegraf.Metric) error {
 	var writeErr error = nil
+
 	for _, metric := range metrics {
 		b, err := f.serializer.Serialize(metric)
 		if err != nil {
 			return fmt.Errorf("failed to serialize message: %s", err)
 		}
 
-		for _, writer := range f.writers {
-			_, err = writer.Write(b)
-			if err != nil && writer != os.Stdout {
-				writeErr = fmt.Errorf("E! failed to write message: %s, %s", b, err)
-			}
+		_, err = f.writer.Write(b)
+		if err != nil && f.writer != os.Stdout {
```

Comment:
You can remove the second half of the check, as it will always be `true`

`fmt.Println(io.MultiWriter([]io.Writer{os.Stdout}...) == os.Stdout)`